pkgs/applications/misc/kplex/default.nix

```
@@ -0,0 +1,31 @@
+{
+  stdenv,
+  fetchurl,
+  lib,
+}:
+stdenv.mkDerivation rec {
+  pname = "kplex";
+  version = "1.4";
+  src = fetchurl {
+    url = "https://github.com/stripydog/kplex/archive/refs/tags/v${version}.tar.gz";
+    hash = "sha256-6+PcgQ6QUx3ID6cJNaiNydsdWxqeJZsWpoitT1sJhCw=";
+  };
+
+  buildPhase = ''
+    make
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp kplex $out/bin
+  '';
```

```suggestion
  installPhase = ''
    install -D -m 0555 kplex $out/bin/kplex
  '';
```
we can use install here which makes file permissions a bit more obvious